components/homme/test_execs/share_kokkos_ut/CMakeLists.txt

```diff
@@ -163,6 +163,47 @@ ELSE()
 ENDIF()
 cxx_unit_test (sphere_op_ut "${SPHERE_OP_UT_F90_SRCS}" "${SPHERE_OP_UT_CXX_SRCS}" "${SPHERE_OP_UT_INCLUDE_DIRS}" "${CONFIG_DEFINES}" ${NUM_CPUS})
 
+### Limiters unit test ###
+
+SET (LIMITERS_UT_F90_SRCS
+  ${SRC_SHARE_DIR}/control_mod.F90
+  ${SRC_SHARE_DIR}/coordinate_systems_mod.F90
+  ${SRC_SHARE_DIR}/derivative_mod_base.F90
+  ${SRC_SHARE_DIR}/dimensions_mod.F90
+  ${SRC_SHARE_DIR}/edgetype_mod.F90
+  ${SRC_SHARE_DIR}/element_mod.F90
+  ${SRC_SHARE_DIR}/gridgraph_mod.F90
+  ${SRC_SHARE_DIR}/kinds.F90
+  ${SRC_SHARE_DIR}/parallel_mod.F90
+  ${SRC_SHARE_DIR}/physical_constants.F90
+  ${SRC_SHARE_DIR}/params_mod.F90
+  ${SRC_SHARE_DIR}/quadrature_mod.F90
+  ${SRC_PREQX_DIR}/element_state.F90
+  ${SHARE_UT_DIR}/limiters_interface.F90
+)
+SET (LIMITERS_UT_CXX_SRCS
+  ${SRC_SHARE_DIR}/cxx/EulerStepFunctorImpl.hpp
+  ${SRC_SHARE_DIR}/cxx/Hommexx_Session.cpp
+  ${SRC_SHARE_DIR}/cxx/mpi/Comm.cpp
+  ${SRC_SHARE_DIR}/cxx/ExecSpaceDefs.cpp
+  ${SHARE_UT_DIR}/limiters.cpp
+)
+
+SET (CONFIG_DEFINES NP=4 NC=4 PLEV=16 QSIZE_D=4 _MPI=1 _PRIM HOMMEXX_NO_CONTEXT HOMMEXX_NO_MPI_CONTEXT)
```

Comment:
For better testing of pack arithmetic, change PLEV to, say, 13.

Reply:
Good point